packages/common/src/hooks/useDownloadTrackButtons.ts

```diff
@@ -78,11 +79,7 @@ const useCurrentStems = ({
   // Sort the stems, filter deletes
   const stemTracks = Object.values(stemTracksMap)
     .filter((t) => !t._marked_deleted && !t.is_delete)
-    .sort(
-      (a, b) =>
-        moment(a.created_at).milliseconds() -
-        moment(b.created_at).milliseconds()
-    )
+    .sort((a, b) => dayjs(a.created_at).diff(dayjs(b.created_at)))
```

Comment:
Might be unnecessary, but we could arguably write a reusable sort comparator `sortByDateAscending` such that you can do `.sort(sortByDateAscending)`. I always find the custom sort comparators difficult to parse because you have to read the docs to remember the rules (`The return value should be a number whose sign indicates the relative order of the two elements: negative if a is less than b, positive if a is greater than b, and zero if they are equal. NaN is treated as 0.`)

Reply:
yeah agree! ty